gulpfile.js

```diff
@@ -216,9 +216,7 @@ function librarySizeTask() {
 
 function moduleSizesTask() {
   return gulp.src(srcDir + '**/*.js')
-    .pipe(uglify({
-      preserveComments: 'some'
-    }))
+    .pipe(uglify())
```

Comment:
Why did you remove `preserveComments: 'some'`, we use it to keep the license header?